cloudferrylib/os/network/neutron.py

```diff
@@ -1207,7 +1207,14 @@ def upload_floatingips(self, networks, src_floats):
         LOG.info("Uploading floating IPs...")
         existing_networks = self.get_networks()
         new_floating_ids = []
+        ipfloatings = {net['floating_ip_address']: net['id']
+                       for net in self.neutron_client.list_floatingips()
```

Comment:
It is little bit confusing, that one element of the `self.neutron_client.list_floatingips()['floatingips']` result is `net` instead of `fip`, `floating`, `floating_ip` etc. Could you rename it please?